Assets/Resources/attackArea.prefab

```diff
@@ -68,6 +68,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 689a34cdfc26dda4ba4f7c983c2aa16b, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  with: enemy
+  damageAmount: 2
```

Comment:
I dont actully know what this change is about, to my knowlege I didnt change anything in attack area.